pkgs/development/libraries/audiofile/default.nix

```diff
@@ -1,4 +1,5 @@
-{ stdenv, lib, fetchurl, fetchpatch, alsaLib, AudioUnit, CoreServices }:
+{ stdenv, lib, fetchurl, fetchpatch, alsaLib, AudioUnit, CoreServices
+, static ? stdenv.targetPlatform.isStatic }:
```

Comment:
The `static` argument as implemented here doesn't actually enable a static build, it only applies a hack.

Reply:
quite right! corrected both instances.